### ISSUE TITLE
About the training hyperparameter setting of yolov_x

### ISSUE THREAD

**Issue by lh4027** [YuHengsss/YOLOV#44]
Hello author, I would appreciate your advice on the setting of training hyperparameters when training the yolov_x large model, do the learning rate and epochs need to be increased?

**Comment by YuHengsss** [comment]
Yep, you should tune it upon your dataset
